### DISCUSSION TITLE
Convert GitHub Pro account to Free Organization with Private Repos

### DISCUSSION THREAD

**Discussion by americloud** [community/community#23329] [New to GitHub]
<p>I have a current paid Pro with multiple private repos. With this new announcement, I would like to convert my account into an organization and assign another account as the owner. When I try to convert, I get this message:</p>
<p>Any private repositories for  <strong>_________</strong>  will be disabled. To maintain access, <a href="https://help.github.com/articles/making-a-private-repository-public/" rel="nofollow noopener">make them public</a> before downgrading.</p>
<p>Please help how I can convert into the free teams organization plan with my private repos. Thanks. </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for reaching out, I’m sorry for the trouble here! We’ve done some testing on our end, and it appears that this error is simply copy that was not updated when these changes were made earlier today.</p>
<p>You should be able to successfully proceed with converting the account to a free organization while keeping those repositories private (and maintaining access to them!).</p>
<p>We’re going to get the incorrect copy updated on our side, and thank you very much for bringing it to our attention.</p>
